Lombiq.Tests.UI/Docs/Migration.md

```diff
@@ -41,58 +41,58 @@ app.Run();
 + }
 ```
 
-### Preparing UI test project
+### Preparing the UI test project
 
 1. Add a _project reference_ of the web app to be tested to the UI test project.
 
-```diff
-  </ItemGroup>
+    ```diff
+    </ItemGroup>
 
-  <ItemGroup>
-+    <ProjectReference Include="..\..\src\Lombiq.OSOCE.Web\Lombiq.OSOCE.Web.csproj" />
-    <ProjectReference Include="..\..\src\Modules\Lombiq.ChartJs\Lombiq.ChartJs.Tests.UI\Lombiq.ChartJs.Tests.UI.csproj" />
-    <ProjectReference Include="..\..\src\Modules\Lombiq.DataTables\Lombiq.DataTables\Tests\Lombiq.DataTables.Tests.UI\Lombiq.DataTables.Tests.UI.csproj" />
-    <ProjectReference Include="..\..\src\Modules\Lombiq.HelpfulExtensions\Lombiq.HelpfulExtensions.Tests.UI\Lombiq.HelpfulExtensions.Tests.UI.csproj" />
-```
+    <ItemGroup>
+    +    <ProjectReference Include="..\..\src\Lombiq.OSOCE.Web\Lombiq.OSOCE.Web.csproj" />
+        <ProjectReference Include="..\..\src\Modules\Lombiq.ChartJs\Lombiq.ChartJs.Tests.UI\Lombiq.ChartJs.Tests.UI.csproj" />
+        <ProjectReference Include="..\..\src\Modules\Lombiq.DataTables\Lombiq.DataTables\Tests\Lombiq.DataTables.Tests.UI\Lombiq.DataTables.Tests.UI.csproj" />
+        <ProjectReference Include="..\..\src\Modules\Lombiq.HelpfulExtensions\Lombiq.HelpfulExtensions.Tests.UI\Lombiq.HelpfulExtensions.Tests.UI.csproj" />
+    ```
 
-2. Change `OrchardCoreUITestBase` implementation like below. `AppAssemblyPath` is not required anymore.
+1. Change `OrchardCoreUITestBase` implementation like below. `AppAssemblyPath` is not required anymore.
```

Comment:
```suggestion
2. Change `OrchardCoreUITestBase` implementation like below. `AppAssemblyPath` is not required anymore.
```
here, or `1.` above(under `### Preparing the web application project.`)

Reply:
Markdown renderers will understand

```
1. asdf
1. asdfasdfa
1. pasdf
```

as an ordered list and render this:

1. asdf
1. asdfasdfa
1. pasdf

However, when looking at the text directly, it's indeed nicer to have the proper numbering (it's just a pain if you have to reorder or insert/delete items).

Reply:
I introduced that change due to a misunderstanding of the markdownlint warning.

I've just verified that consecutive numbering indeed does **not** trigger any warning.